docker/docker-compose.yml

```diff
@@ -103,6 +103,8 @@ services:
       POSTGRES_DB: ${MF_USERS_DB}
     networks:
       - mainflux-base-net
+    ports:
+      - "6001:5432"
```

Comment:
Can you maybe use ENVARS here? It's hard to understand this values.

Reply:
this is mistake, will remove this, only for development purposes